common/spring-cloud-stream-modules-common-configuration/src/main/java/org/springframework/cloud/stream/module/file/remote/AbstractRemoteFileProperties.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.stream.module.file.remote;
+
+import org.hibernate.validator.constraints.NotBlank;
+
+/**
+ * @author Gary Russell
+ *
+ */
+public abstract class AbstractRemoteFileProperties {
```

Comment:
I like more when we mark them all with the `@ConfigurationProperties`, don't inherit and just specify in the `EnableConfigurationProperties`.
See the #46 for similar problem around the common properties.

I won't mind if it doesn't matter though...